lib/classes/Variables.js

```diff
@@ -889,7 +889,7 @@ class Variables {
     if (
       valueToPopulate === null ||
       typeof valueToPopulate === 'undefined' ||
-      (typeof valueToPopulate === 'object' && _.isEmpty(valueToPopulate))
+      (!Array.isArray(valueToPopulate) && _.isEmpty(valueToPopulate))
```

Comment:
I've just noticed, that we also should confirm that we deal with an object,
as otherwise this condition will resolve as truthy for any primitive (strings, numbers, booleans), and so far it wasn't.

Reply:
yep sorry, fixed